packages/__tests__/src/rounding.ts

```diff
@@ -0,0 +1,24 @@
+export function round(z = 8): (n: number) => number {
+    const p = 10 ** z;
+    return (n: number): number => Math.round(n * p) / p;
+}
+
+export function roundJson(json: any, z = 8): any {
```

Comment:
why json?  you dont `JSON.parse()` it?

Reply:
Meaning Plain old javascript objects; not classes or functions etc.

Reply:
If i see `json: any` it normally implies the `any` is a string

could be `roundObject` Also not a huge fan of the any's especially not returning the any

I wonder if you could type it with something like `roundObject<T extends Object>(obj:T, z =8): T { ... }`

Reply:
Is can round non objects too. How about `export function roundAnything(thing: any, z = 8): any {`?

Reply:
or `round` and make the other one `roundNumber` or `roundFunc`

Reply:
Now makeRound and round